src/pages/docs/infrastructure/accounts/google-cloud/index.md

```diff
@@ -8,18 +8,45 @@ navOrder: 30
 ---
 
 :::div{.hint}
-Google Cloud Accounts were added in Octopus **2021.2**.
+Google Cloud Accounts were added in Octopus **2021.2**, Generic OpenId Connect Accounts were added in **2024.4**
 :::
 
-To deploy infrastructure to Google Cloud Platform, you can define a Google cloud account in Octopus.
+To deploy infrastructure to Google Cloud Platform, you can define a Google cloud or Generic OpenId Connect account in Octopus.
 
-Octopus manages the GCP credentials used by the Google cloud steps.
+The Generic OpenId Connect Account generates a JWT that can be used for [OpenID Connect](/docs/infrastructure/openid-connect.md) authentication. The Google cloud account uses the JSON key file credentials that can be retrieved from the service account assigned to the instance that is executing the deployment.
 
-The Google cloud account is the JSON key file credentials that can be retrieved from the service account assigned to the instance that is executing the deployment.
+## Generic OpenId Connect Account
+Google Cloud steps can use a Generic OpenId Connect Account for authentication.
+
+1. Navigate to **Infrastructure ➜ Accounts**, click the **ADD ACCOUNT** and select **Generic Oidc Account**.
+1. Add a memorable name for the account.
+1. Set the [Deployments and Runbooks](/docs/infrastructure/openid-connect.md) subject generator
+1. set an audience, this should match the audience set on the Workload Identity Federation. By default this is `https://iam.googleapis.com/projects/{project-id}/locations/global/workloadIdentityPools/{pool-id}/providers/{provider-id}`
+1. Click the **SAVE**, to test the account set it as the account on a gcloud script step.
+
+See the [Google cloud documentation](https://cloud.google.com/iam/docs/workload-identity-federation-with-other-providers) for instructions on creating and configuring a Workload Identity Federation. To authenticate octopus calls the gcloud cli
```

Comment:
Isn't this done for the user already, as per https://github.com/OctopusDeploy/Calamari/pull/1399?

Reply:
Yes, in that section I'm just trying to explain the authentication process behind the scenes

Reply:
It felt like that was what the user needed to do, particularly if you don't read it closely.

Reply:
I've updated that message, if you think it's better to just remove the snippet all together let me know.